osversion/windowsbuilds.go

```diff
@@ -1,10 +1,23 @@
 package osversion
 
 const (
-
-	// RS2 was a client-only release in case you're asking why it's not in the list.
+	// RS1 (version 1607, codename "Redstone 1") corresponds to Windows Server
+	// 2016 (ltsc2016) and Windows 10 (Anniversary Update).
 	RS1 = 14393
+
+	// RS2 (version 1703, codename "Redstone 2") was a client-only update, and
+	// corresponds to Windows 10 (Creators Update).
+	RS2 = 15063
```

Comment:
FWIW, I added RS2, even though it won't be used on Windows server; this package could still be useful to get the right version numbers (and I thought it would not be a big issue to have the additional constant)

Reply:
I wouldn't. `RS2` is entirely out of support